### DISCUSSION TITLE
Yarpviz imported in yarp

### DISCUSSION THREAD

**Discussion by Nicogene** [robotology/community#261] [Announcements]
[Yarpviz](https://github.com/robotology/yarpviz) has been imported in `YARP` on `devel` branch. 
It will be available on master from release **2.3.72**.
More information on how to compile it inside `YARP` can be found [here](http://www.yarp.it/devel/yarpviz.html).

Enjoy :wink:

**Comment by pattacini** [comment]
One day we should provide an easy way to install `graphiz` on Windows x64, possibly through `vcpkg`.

**Comment by traversaro** [comment]
Relative vcpkg issue: https://github.com/Microsoft/vcpkg/issues/737 .
